install_dependencies_choco.ps1

```
@@ -0,0 +1,87 @@
+
+# Check if Chocolatey is installed
+Write-Host "Checking for Chocolatey..." -ForegroundColor Yellow
+$choco = Get-Command choco -ErrorAction SilentlyContinue
+
+if (-not $choco) {
+    Write-Host "❌ Chocolatey not found!" -ForegroundColor Red
+    Write-Host ""
+    Write-Host "Installing Chocolatey..." -ForegroundColor Yellow
+    
+    [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072
+    iex ((New-Object System.Net.WebClient).DownloadString('https://community.chocolatey.org/install.ps1'))
```

PSScriptAnalyzer (1.25.0)

[warning] Invoke-Expression is used. Please remove Invoke-Expression from script and find other options instead.

(PSAvoidUsingInvokeExpression)